spec/rails/doc/smart/expected.yaml

```diff
@@ -83,49 +65,8 @@ paths:
                 properties:
                   message:
                     type: string
-                  no_such_field:
-                    type: string
-                    example: This field does not exist in rspec
```

Comment:
Note) `no_suche_field` was removed because it matched the pattern `7. paths.*.*.responses.*.content.application/json.schema.properties.*`